### PR TITLE
Standardize compare number

### PR DESCRIPTION
**Pull request by frzyc**
## Describe your changes
When comparing builds, the "white text" should always be the focused build, and the "delta" colored text will be the diff between the focused/compare build. This prevents the "white text" from changing regardless of the compare state, which should give a more "stable" experience to compare numbers for users.

## Issue or discord link

- Resolves https://github.com/frzyc/genshin-optimizer/issues/1817
## Testing/validation

For screenshot, Left is new, Right is production
Overview screen 
Screenshot 2024-03-25 011002
Theorycraft
Screenshot 2024-03-25 010707
Generated Build
Screenshot 2024-03-25 011418
Compare build (swapping single artifact)
Screenshot 2024-03-25 012342

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

**Comment by nguyentvan7** [comment]
multi-targets are showing up as red for some reason. Not showing up on beta build 
image

**Comment by nguyentvan7** [comment]
This was already an issue before, but since we are touching this code now:

Should we add a tooltip to the diff value that shows the raw comparison value?
For example, if my current build is 1k dmg
then my compared-to build is 900 dmg

UI will show:
1000 (+100)(+%)

What if we added a tooltip to the `(+100)(+%)` portion that will show `Compared to: 900` or something similar?

To me, without prior knowledge of which "direction" the comparison goes, it is hard to know if the diff value is `current - compare` or `compare - current`

**Comment by frzyc** [comment]
> This was already an issue before, but since we are touching this code now:
> 
> Should we add a tooltip to the diff value that shows the raw comparison value? For example, if my current build is 1k dmg then my compared-to build is 900 dmg
> 
> UI will show: 1000 (+100)(+%)
> 
> What if we added a tooltip to the `(+100)(+%)` portion that will show `Compared to: 900` or something similar?
> 
> To me, without prior knowledge of which "direction" the comparison goes, it is hard to know if the diff value is `current - compare` or `compare - current`

Sure, sounds like we can add it to see how people perceive it, and we can decide what to do next.

File: apps/frontend/src/app/Components/Character/CharacterEditor/index.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ function CharacterEditorContent({
     return {
       data: charUIData,
       teamData,
-      oldData: undefined,
```
Comment (nguyentvan7) [inline apps/frontend/src/app/Components/Character/CharacterEditor/index.tsx:63]: good change

File: apps/frontend/src/app/Components/FieldDisplay.tsx [hunk 2]
```diff
@@ -121,25 +124,49 @@ export function NodeFieldDisplay({
...
     fieldVal = (
       <>
-        <span>{valueString(oldValue, node.info.unit, node.info.fixed)}</span>
+        <span>{valueString(nodeValue, node.info.unit, node.info.fixed)}</span>
         {Math.abs(diff) > 0.0001 && (
-          <>
-            <ColorText color={diff > 0 ? 'success' : 'error'}>
-              {diff > 0 ? '+' : ''}
-              {valueString(diff, node.info.unit, node.info.fixed)}
+          <Tooltip
```
Comment (nguyentvan7) [inline apps/frontend/src/app/Components/FieldDisplay.tsx:136]: This tooltip has bad contrast
image
Can we use bootstraptooltip instead for the black background, and to match our other tooltips?

File: apps/frontend/src/app/Components/FieldDisplay.tsx [hunk 3]
```diff
@@ -121,25 +124,49 @@ export function NodeFieldDisplay({
...
+            title={
+              <Typography>
+                Compare to{' '}
+                <ColorText color={diff > 0 ? 'success' : 'error'}>
```
Comment (nguyentvan7) [inline apps/frontend/src/app/Components/FieldDisplay.tsx:140]: image
This coloring is a little confusing honestly. Might be better without any coloring?

Like it doesn't seem right that the lower number in the tooltip is green for some reason, as if indicating it is better. Im not sure if there would be similar confusion when its flipped

**Review by nguyentvan7 — APPROVED** [review]

**Pull request merged**